baseline-config.json

```diff
@@ -0,0 +1,39 @@
+{
+    // Set to `true` to reindent the HTML output files. Set to `false` to generate output files more
+    // quickly, but with extra whitespace.
+    "beautify": true,
+    // Components to display in each HTML output file.
+    "components": {
+        // Set to `true` to display a summary section at the top of each page. Set to `false` to omit
+        // the summary section.
+        "summary": true
+    },
+    // Maps CSS class names used in the template files to class names used in the output files.
+    // IMPORTANT: The CSS class names in the template files will change. For now, don't use this
+    // option.
+    //"cssClassMap": "",
+    // The character encoding to write to the <meta> tag in HTML output files. You are strongly
+    // encouraged to use the default setting.
+    "encoding": "utf-8",
+    // The locale for displaying localized text.
+    "locale": "en",
+    // Set to `true` to convert text from Markdown to HTML or `false` to disable Markdown support.
+    "markdown": true,
+    "sourceFiles": {
+        // Set to `true` to generate pretty-printed HTML versions of the source files, and to link to
+        // the pretty-printed files from the documentation. Set to `false` to omit the pretty-printed
+        // source files and the links to these files.
+        "generate": false,
+        // Set to `true` to include a single link to the source file in the documentation, near the
+        // top of the page. Set to `false` to include a source-file link with each symbol in the
+        // documentation.
+        "singleLink": true
+    },
+    "tables": {
+        // Set to `true` to embed child properties of parameters within the same table row as the
+        // parameter. Set to `false` to list child properties on their own rows.
+        "nestedPropertyTables": true
+    },
+    // Set to 'true' to enable Globals section
```

Comment:
Usually, no, since they're not part of the spec, but in this case they're fine.

Presumably either JSDoc itself or the Baseline theme is using strip-json-comments or similar to avoid parser errors here.

I'll remove the comments both from here and from my related Thorn PR (https://github.com/sugarcrm/thorn/pull/156) if you feel we should get rid of them.